.github/workflows/ci.yml

```diff
@@ -94,8 +98,8 @@ jobs:
           java -jar ./checkstyle.jar -c ./style/checkStyle.xml -f xml ./eventmesh-* \
                | ./reviewdog -f=checkstyle -reporter=github-pr-check -filter-mode=added -fail-on-error
 
-  license-check:
-    name: License Check
+  license-header-check:
+    name: License header Check
```

Comment:
Would you like to help to change the uses repo to main, like here, https://github.com/marketplace/actions/license-eye